MetaMask/metamask-extension
# feat: Adding initial draft of Skeleton component to component-library

## **Description**
Initial draft of the `Skeleton` component to review with the @MetaMask/design-system-engineers based on the findings in the [audit and component discovery sync](https://www.figma.com/file/bA81b0VvoGRgNS4E6059Pa/Loaders%2FSkeleton-Audit-and-Insight-Report?type=whiteboard&node-id=117%3A26&t=AnzMgSbVjzePPMmA-1)

## **Related issues**

Fixes: #20874 

## **Manual testing steps**

1. Go to the latest storybook build on this PR
2. Search `Skeleton` 
3. Check the docs, stories and controls

## **Screenshots/Recordings**

### **Before**

N/A

### **After**

https://github.com/MetaMask/metamask-extension/assets/8112138/612414f0-6da8-40db-af46-d6adad873762

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
